### DISCUSSION TITLE
How to use custom tags with bundles

### DISCUSSION THREAD

**Discussion by freshcot** [werf/werf#6363] [General]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Version

1.2.328

### How to reproduce

When I'm using`WERF_ADD_CUSTOM_TAG_1` and `WERF_TAG` for bundle, `werf` build the image and add an alias for the image (custom tag). But this tag doesn't apper inside values of the bundle, it goes with defalt tag

1. Add env variables
                `WERF_ADD_CUSTOM_TAG_1="${VERSION}-my-api"`
                `WERF_TAG="${VERSION}"`
2. Publish bundle
`werf bundle publish`
3. Download the bundle and open values.yaml

### Result

You will see default tag for the image

### Expected result

Image tag inside values.yaml should content custom tag

**Comment by alexey-igrychev** [comment]
If you want to use custom tags in bundles, you need to enable the `WERF_USE_CUSTOM_TAG` option. You can specify any number of `WERF_ADD_CUSTOM_TAG` options, and they will not affect the main tag.​
